.github/workflows/docs-impact-review.yml

```
@@ -48,14 +42,15 @@ jobs:
           sparse-checkout-cone-mode: false
 
       - name: Analyze documentation impact
+        id: docs-analysis
+        if: ${{ secrets.ANTHROPIC_API_KEY != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)